eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/UtxoSpec.hs

```diff
@@ -64,6 +67,49 @@ spec =
         [fromNativeScript spendingScript, fromNativeScript spendingScript]
           ++ extraScripts
           ++ extraScripts
+
+    it "deregistering returns the deposit" $ do
```

Comment:
Is this the right place for this test?  I know it's not very precise how we assign tests to specs, especially when not testing for a specific predicate failure, but I find it surprising to see it in `UtxoSpec` . 
`CertSpec`, `DelegSpec` or `EpochSpec` would seem less surprising for me, but perhaps there is a reason that I'm missing.

Reply:
I moved it to `DelegSpec`